package.json

```
@@ -149,6 +149,13 @@
       "packages/pkg-tests"
     ]
   },
+  "keywords": [
+    "yarn",
+    "packages",
+    "package-manager",
+    "npm",
+    "JavaScript"
+]
```

It would be nice to sort the keywords and keep them lowercase.
Also, I would add those keywords as well: `dependency` and `cli`.

Here is a suggestion:
```suggestion
"keywords": [
    "cli",
    "dependency",
    "javascript",
    "npm",
    "package-manager",
    "packages",
    "yarn"
]
```